compiler-rt/lib/sanitizer_common/sanitizer_deadlock_detector.h

```diff
@@ -395,14 +395,14 @@ class DeadlockDetector {
     int unique_tid;
   };
 
-  uptr current_epoch_;
+  usize current_epoch_;
```

Comment:
FWIW, I changed all the indices, etc. in this file to also be usize.  The only thing in this file that holds actual pointers is the `data_` array.